Formula/bun.rb

```diff
@@ -0,0 +1,15 @@
+class Bun < Formula
+  desc "Bun is a fast all-in-one JavaScript runtime"
+  homepage "https://bun.sh"
+  url "https://github.com/Jarred-Sumner/bun-releases-for-updater/releases/latest/download/bun-linux-x64.zip"
+  sha256 "51c04d71627bf99868527f6a00397aba8bc013a90e1ef1a02080a4e93d291c0d"
+  license "MIT"
+
+  def install
+    bin.install "bun"
+  end
+
+  test do
+    system "false"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.